src/common/components/select-repository-list/index.js

```diff
@@ -102,37 +102,32 @@ export class SelectRepositoryListComponent extends Component {
     this.props.dispatch(fetchUserRepositories(pageNumber));
   }
 
-  pageSlice(array, pageLinks) {
+  pageSlice(repositoriesIndex, pageLinks) {
```

Comment:
This method seems quite independent from SelectRepositoryList (it doesn't seem to have any reference to `this`), so maybe it could actually be external helper function?

Would that help in any way? Make it clearer? Easier to test? - if not, never mind ;)